pytest_fixtures/rh_cloud.py

```diff
@@ -1,50 +1,88 @@
 import pytest
-from nailgun import entities
+from broker.broker import VMBroker
 
 from robottelo.config import settings
 from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.constants import DISTRO_RHEL7
 from robottelo.constants import DISTRO_RHEL8
+from robottelo.helpers import file_downloader
+
+
+@pytest.fixture(scope='module')
+def rhcloud_sat_host(satellite_factory):
+    """A module level fixture that provides a Satellite based on config settings"""
+    new_sat = satellite_factory()
+    yield new_sat
+    VMBroker(hosts=[new_sat]).checkin()
+
+
+@pytest.fixture(scope='module')
+def rhcloud_manifest_org(rhcloud_sat_host):
```

Comment:
you could use `satellite_host` here instead

https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/broker.py#L52

Reply:
That's a function level fixture and I am using module level fixture to reduce test time.